docs/user-flows.md

```diff
@@ -182,15 +175,13 @@ async function search(page) {
   // Setup the browser and Lighthouse.
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
-  const flow = await lighthouse.startFlow(page);
+  const flow = await startFlow(page);
 
   // Phase 1 - Navigate to the landing page.
   await flow.navigate('https://www.bestbuy.com');
 
   // Phase 2 - Interact with the page and submit the search form.
-  await flow.startTimespan();
-  await search(page);
-  await flow.endTimespan();
+  await flow.navigate(async () => await search(page));
```

Comment:
This is replacing a timespan with a navigation, can you revert? `startTimespan`/`endTimespan` still work.